### DISCUSSION TITLE
Send change values via Telnet from loxone

### DISCUSSION THREAD

**Discussion by DiabloVmax1200** [Arnold-n/P1P2MQTT#120] [Q&A] [answered]
Is it possible to steer the Daikin via telnet commands? I ask because I ve read in some discussions that only one telnet session is possible. So I will get in trouble if I have to do something for maintenance via telnet?

**Comment by Arnold-n** [comment]
That should work - you can enter commands via the (single) telnet session in parallel with commands given via (HA/)MQTT.

**Reply by DiabloVmax1200** [reply]
OK, but will it also work if I send e.g. every 5 min the room temperature value via Telnet via Loxone. And additionally start a telnet session from PC to do an update of the device - or debug something?

**Reply by Arnold-n** [reply, marked as answer]
Indeed that would not work - you can only have one telnet session. This is a current limitation of the telnet library. But you can give any commands via MQTT P1P2/W yourself, and track responses via P1P2/S, to avoid such a conflict.
During a device update no commands can be given.

**Reply by DiabloVmax1200** [reply]
Ah! I had the misunderstanding that I couldn't write the room temperature via P1P2/W but this (F1 xxx) works as well. Thanks! Then this is for sure the preferred way to do it!
